PhoneBook.Doc415/EmailSender.cs

```diff
@@ -0,0 +1,58 @@
+﻿using PhoneBook.Doc415.Models;
+using Spectre.Console;
+using System.Net;
+using System.Net.Mail;
+namespace PhoneBook.Doc415;
+
+internal class EmailSender
+{
+    string smtpServer;
+    string userName;
+    string password;
+    string subject;
+    string body;
+    string recipentEmail;
+    string myEmail;
+    public void SetUpSender()
+    {
+        smtpServer = AnsiConsole.Ask<string>("Enter your smtp server (smtp.gmail.com) (alpha stage no validation yet): "); //add validation
+        userName = AnsiConsole.Ask<string>("Enter user name: ");
+        password = AnsiConsole.Ask<string>("Enter password: ");
+        
+        do
+        {
+            myEmail = AnsiConsole.Ask<string>("Enter your e-mail adress (foo@foo.com):  ");
+            
+        } while (!Validators.isValidEmail(myEmail));
+        
+    }
+
+    public void SendEmail(Contact contact)
```

Comment:
I am missing tests for new code

Reply:
Do you mean you have failing tests for the code? 
I dont have knowledge about tests yet, like unit tesys ect.

Reply:
I mean that you have not written any tests for this piece of changes (unit / acceptance / integration)